roles/ceph-common/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ ceph:
   #  - sdd
   #bcache_ssd_device: sdg 
 
-  stable_release: hammer
+  stable_release: jewel
```

Comment:
Will the apt-get source url also use the pack version?
like 
`deb https://apt-mirror.openstack.blueboxgrid.com/ceph/debian-10.2.2/ trusty main`